### PR TITLE
Sw 641 add range slider to axiom

### PR DESCRIPTION
**Pull request by clairebw**
This adds a Range slider with two handles.
Screen Shot 2019-04-08 at 15 47 50

### CODE REVIEW

File: packages/axiom-components/src/Slider/Range.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    if (values[0] > values[1]) {
+      switch (this.focusedHandleIndex) {
+      case 0:
+        values[0] = values[1];
+        break;
+      case 1:
+        values[1] = values[0];
+        break;
+      default:
+        return;
```
Comment (lpoulter) [inline packages/axiom-components/src/Slider/Range.js]: Might be nice to throw/log an error here.

File: packages/axiom-components/src/Slider/Range.js [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+    this.onChange(this.getValues(clientX));
+  }
+
+  handleKeyDown(event) {
+    const { step } = this.props;
+    const { values } = this.state;
+
+    switch (event.key) {
+    case 'ArrowLeft':
```
Comment (lpoulter) [inline packages/axiom-components/src/Slider/Range.js]: Formatting looks strange here should `case` be indented?

**Comment by lpoulter** [comment]
Hi @clairebw thanks for this. Looks like there are some lint and test issues on the build. Are you able to loop in design and link the new component in the docs once it build?

File: packages/axiom-components/src/Slider/Range.js (hunk 2 above)
Comment (clairebw) [reply packages/axiom-components/src/Slider/Range.js]: The eslint rule explicitly requires that there is no indentation for `switch` cases: https://github.com/clairebw/axiom-react/blob/04ef2645fbce630b80043fd6539262c27ddf348a/packages/eslint-config-axiom/src/eslintrc.json#L18

**Comment by clairebw** [comment]
@lpoulter @TomWija @MariamBashir Thanks for your reviews. I fixed the tests, addressed the comments and @lilybrandwatch also approved it. Would you mind having a second look?

**Pull request merged**